exercise-book/src/tcp-server.md

```diff
@@ -0,0 +1,81 @@
+# Interactive TCP Echo Server
+
+In this exercise, we will make a simple TCP "echo" server using APIs in Rust standard Library. Here's how an interaction with it would look like from a client point of view. You connect to it using `nc`, for example:
```

Comment:
```suggestion
In this exercise, we will make a simple TCP "echo" server using APIs in Rust's Standard Library.

Here's how an interaction with it would look like from a client point of view. You connect to it using `nc`, for example:
```